.coveragerc

```diff
@@ -2,9 +2,9 @@
 branch = True
 omit =
     # omit all tests
-    src/alchemtest/tests/*
+    alchemtest/tests/*
     # omit the versioneer-installed _version.py
-    src/alchemtest/_version.py
+    alchemtest/_version.py
```

Comment:
Do we still have a file `_version.py`?

Reply:
It is automatically generated by git versioneer when you pip install it. Looks like
`__version__ = "0.10.0.dev0"`